apache-httpd.conf

```diff
@@ -0,0 +1,544 @@
+#
```

Comment:
It thought it would be better too keep the file close to the original. But yes I can remove them